RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -3347,7 +3355,17 @@ boolean L11_defeatEd()
 	auto_log_info("Time to waste all of Ed's Ka Coins :(", "blue");
 
 	set_property("auto_nextEncounter","Ed the Undying");
-	autoAdv($location[The Lower Chambers]);
+	if(in_bhy())
+	{
+		maximize("beeosity,-tie", false);
+		set_property("auto_disableAdventureHandling", true);
+		autoAdvBypass("place.php?whichplace=pyramid&action=pyramid_state1a", $location[The Lower Chambers]);
+		set_property("auto_disableAdventureHandling", false);
+	}
+	else
+	{
+		autoAdv($location[The Lower Chambers]);
+	}
 	if(in_pokefam() || in_koe())
```

Comment:
Maximizing for only beeosity results in a B count of 0, which is basically required to survive the 5 round fight with Ed.

Reply:
There might be cleaner way to do this reliably, but auto_pre_adv is difficult to understand, and adding it here seemed more reliable longterm.